packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -277,7 +277,7 @@ export const components: ThemeOptions["components"] = {
         lineHeight: "1.14285714",
         whiteSpace: "nowrap",
 
-        "& + &": {
+        ".MuiButton-root + &": {
```

Comment:
I feel like this could totally break at any point in time. I get that it's not your HTML.

Reply:
Yeah, this isn't especially safe. Once we have `<ButtonGroup>` done, we can remove this sibling styling altogether.

Alternately, I can see if there's a `MuiButtonClasses` to pull from.